### PR TITLE
Codebar and a few changes to past assignments. 

### PR DESCRIPTION
**Pull request by Jonrosario5**

### CODE REVIEW

**Comment by DaltonHart** [comment]
Looks great. make sure to turn in your flask api assignment.

File: Jon-Jonrosario5/Week-8/state-capitals/capitals.py [hunk 1]
```diff
@@ -167,6 +168,7 @@
 
 prompt = '> '
```
Comment (DaltonHart) [inline Jon-Jonrosario5/Week-8/state-capitals/capitals.py]: Not sure why this is here. Reasoning?

Comment (Jonrosario5) [reply Jon-Jonrosario5/Week-8/state-capitals/capitals.py:169]: It is more for a user friendly experience in the terminal. When prompted to answer a question, the ">" appears, it's kinda like a signal for the user to type something. A small add that wasn't necessary but a nice "Jon" touch.

Comment (Jonrosario5) [reply Jon-Jonrosario5/Week-8/state-capitals/capitals.py:169]: @DaltonHart

**Pull request merged**